core/src/executor/evaluate/function.rs

```diff
@@ -223,6 +223,35 @@ pub fn substr(
     Ok(Value::Str(string))
 }
 
+pub fn ascii(name: String, expr: Evaluated<'_>) -> Result<Value> {
+    let string = eval_to_str!(name, expr);
+
+    if string.len() != 1 {
+        return Err(EvaluateError::AsciiFunctionRequiresSingleCharacterValue.into());
+    }
+
+    let char = string.chars().next();
```

Comment:
`char` is the name of a primitive type, so we recommend something else.
ex) `character`